src/globals.c

```diff
@@ -1,3 +1,4 @@
+#pragma once
```

Comment:
It is a bit weird to place `#pragma once` in a non-header file.

Reply:
I don't think current implementation (both MakeFile target script and shecc itself) is capable of linking source file to header file, in other words, we cannot declare global variables and function prototypes in `globals.h` and include it in other files. Also, even after migrating to header file, the source file would still need to add pragma since again, shecc is not capable of linking which requires to includes source and header file at once. This is workaround should be ignored by gcc when compiling, and is required for shecc in order to not included multiple times.